client/src/components/OfflineWarning/offline-warning.tsx

```diff
@@ -42,10 +41,10 @@ function OfflineWarning({
   }
 
   return showWarning ? (
-    <>
+    <div data-playwright-test-label='offline-warning'>
       <div className='offline-warning alert-info'>{message}</div>
```

Comment:
```suggestion
       <div className='offline-warning alert-info'  data-playwright-test-label='offline-warning' >{message}</div>
```
We don't wanna add a specific div for identifying the element.